### PR TITLE
add(Adapter,MergedCalendarActivity,MergedCalendarListResponse,MergedC…

### PR DESCRIPTION
**Pull request by RedRiot**
…alendarResponse) and modify(Manifest and ChooseAccountActivity)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
         debug {
-            buildConfigField "String", "LOCAL_IP_ADDRESS", System.getenv("LOCAL_IP_ADDRESS")
+           buildConfigField "String", "LOCAL_IP_ADDRESS", System.getenv("LOCAL_IP_ADDRESS")
```
Comment (adamgyulavari) [inline app/build.gradle]: @RedRiot indentation

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -26,6 +26,7 @@
         </activity>
         <activity android:name=".SelectCalendarActivity" />
         <activity android:name=".ChooseAccountActivity"/>
+        <activity android:name=".MergedCalendarActivity"/>
     </application>
 
 </manifest>
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @RedRiot new line

File: app/src/main/java/com/greenfox/kalendaryo/MergedCalendarActivity.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                MergedCalendarListResponse mergedCalendarListResponse = response.body();
+                adapter.addAll(mergedCalendarListResponse.getMergedCalendars());
+
+            }
+
+            @Override
+            public void onFailure(Call<MergedCalendarListResponse> call, Throwable t) {
+                t.printStackTrace();
+            }
+        });
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/MergedCalendarActivity.java]: @RedRiot this should be in `MergedKalendarFragment` in `MainActivity`

File: app/src/main/java/com/greenfox/kalendaryo/ChooseAccountActivity.java [hunk 4]
```diff
@@ -67,6 +67,7 @@ public void onFailure(Call<MergedKalendarResponse> call, Throwable t) {
                 });
 
                 Intent i = new Intent(ChooseAccountActivity.this, MainActivity.class);
+                kalpref.setClienttoken(clientToken);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/ChooseAccountActivity.java]: @RedRiot this is unnecessary

File: app/src/main/java/com/greenfox/kalendaryo/adapter/MergedKalendarAdapter.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    public void addMergedCalendarResponse(List<MergedCalendarResponse>  newMergedCalendarResponses) {
+        this.mergedCalendarResponses.addAll(newMergedCalendarResponses);
+        notifyDataSetChanged();
+    }
+
+
+    @Override
+    public MergedKalendarAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.onemergedcal, parent, false);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/adapter/MergedKalendarAdapter.java]: @RedRiot `merged_calendar_item.xml`

File: app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java [hunk 6]
```diff
@@ -4,23 +4,50 @@
...
     FloatingActionButton floatingActionButton;
+    private MergedKalendarAdapter adapter;
+    private RecyclerView recyclerView;
+    private BackendApiService backendApiService;
+
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.kalendarlist, container, false);
+        adapter = new MergedKalendarAdapter(view.getContext());
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java]: @RedRiot `getActivity()`

File: app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java [hunk 7]
```diff
@@ -4,23 +4,50 @@
...
         floatingActionButton = view.findViewById(R.id.choosecalendar);
+        recyclerView = view.findViewById(R.id.apilistcalendars);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java]: @RedRiot `calendar_list`

File: app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java [hunk 8]
```diff
@@ -4,23 +4,50 @@
...
+        KalPref kalPref = new KalPref(this.getContext());
+        String clientToken = kalPref.clientToken();
+        getCalendarResponse(clientToken);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java]: @RedRiot these two can be in one line, the `clientToken` variable is unnecessary

File: app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java [hunk 9]
```diff
@@ -33,4 +60,21 @@ public void onClick(View v) {
...
+                recyclerView.setAdapter(adapter);
+            }
+
+            @Override
+            public void onFailure(Call<MergedCalendarListResponse> call, Throwable t) {
+                t.printStackTrace();
+            }
+        });
+    }
 }
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java]: @RedRiot new line

File: app/src/main/AndroidManifest.xml [hunk 10]
```diff
@@ -16,16 +16,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".LoginActivity"/>
-        <activity android:name=".MainActivity" >
+        <activity android:name=".LoginActivity" />
```
Comment (BalazsSalfay) [inline app/src/main/AndroidManifest.xml:19]: @RedRiot delete space

File: app/src/main/java/com/greenfox/kalendaryo/adapter/MergedKalendarAdapter.java [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+
+        public ViewHolder(View itemView) {
+            super(itemView);
+            mergedCalendarName = itemView.findViewById(R.id.mergedcalendarname);
+            calendarDescription = itemView.findViewById(R.id.mergedcalendardescription);
+        }
+    }
+
+
```
Comment (BalazsSalfay) [inline app/src/main/java/com/greenfox/kalendaryo/adapter/MergedKalendarAdapter.java]: @RedRiot delete these lines

**Review by matheb — APPROVED** [review]

**Pull request merged**